templates/nsca.erb

```diff
@@ -1,5 +1,10 @@
 [monitoring]
-OK       = printf "<%= @send_as_host -%>;%(event);0;%(message)\n" |/usr/sbin/send_nsca -H <%= @send_to_host -%> -d ";"
-WARNING  = printf "<%= @send_as_host -%>;%(event);1;%(message)\n" |/usr/sbin/send_nsca -H <%= @send_to_host -%> -d ";"
-CRITICAL = printf "<%= @send_as_host -%>;%(event);2;%(message)\n" |/usr/sbin/send_nsca -H <%= @send_to_host -%> -d ";"
-UNKNOWN  = printf "<%= @send_as_host -%>;%(event);3;%(message)\n" |/usr/sbin/send_nsca -H <%= @send_to_host -%> -d ";"
+<% if @send_to_host.is_a?(String) -%>
```

Comment:
You might want to omit all of these statements, if `send_to_host` is empty or undefined, etc.